<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>651</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image Filter based on OpenCL(mikewolf_gkd@163.com)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/new/prefix1/images/insert-image.png</selectedon>
   </iconset>
  </property>
  <widget class="QGroupBox" name="grbSetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>951</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting</string>
   </property>
   <widget class="QRadioButton" name="rdbHostcpu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Host CPU</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbDevicecpu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device CPU</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbDevicegpu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device GPU</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkTiming">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbIterates">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblIteraters">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteraters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFiltrparams">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>filter params:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWindowsize">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>window size(n*n):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbIterates_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpecialparams">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>special params:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grbPerfinfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>951</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Performance information</string>
   </property>
   <widget class="QLabel" name="lblPlatform">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PlatForm:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDevice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlatformname">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDevicename">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotaltime">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TotalTime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotaltimevalue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblH2Dtime">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H2DTime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblD2Htime">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>D2HTime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKernaltime">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KernalTime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblH2Dtimevalue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblD2Htimevalue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKernalexectime">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KernalExecTime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKernaltimevalue">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKernalexectimevalue">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBandwidth">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BandWidth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBandwidthvalue">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="tlbSrcimg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDstimg">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="grbImage">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>130</y>
     <width>101</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Image</string>
   </property>
   <widget class="QLabel" name="lblImgwidth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImgwidthvalue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblImgheight">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImgheightvalue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dbsScaled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lblScaled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaled:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
